spec/requests/auth/slack_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
```

Comment:
Disregard, I was thinking of the incorrect framework. You're good to match with Rail naming here